.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
```

Comment:
I prefer "on release" here, but whatever @flying-sheep is happy with

Reply:
since this is blocking someone, let’s do this and refine later